### ISSUE TITLE
Radial Gauge: add option to adjust bar width

### ISSUE THREAD

**Issue by johndoh** [ScottPlot/ScottPlot#2128]
## Feature Suggestion

Please consider adding an option to allow the width of bar in of Radial Chart in [Single Gauge Mode](https://scottplot.net/cookbook/4.1/category/plottable-radialgauge/#single-gauge-mode) to be adjusted. Specifically I'd like to be able to make the bar wider when the chart is shown in single gauge mode. Alternatively have the bar rendered 2 or 2 times the current width when in single gauge mode.

```cs
var plt = new ScottPlot.Plot(600, 400);

double[] values = { 100, 80, 65, 45 };

var gauges = plt.AddRadialGauge(values);
gauges.GaugeMode = ScottPlot.RadialGaugeMode.SingleGauge;
gauges.MaximumAngle = 180;
gauges.StartingAngle = 180;

gauges.Width = 50; // new option

plt.SaveFig("radialgauge_single.png");
```

**Comment by arthurits** [comment]
Hi @johndoh,

Currently, the width of the gauge is computed here:
https://github.com/ScottPlot/ScottPlot/blob/07ab190973cbf8697ff8bdc98b4e6475ce40ea03/src/ScottPlot4/ScottPlot/Plottable/RadialGaugePlot.cs#L267

The more "categories" (higher `GaugeCount`), the thinner the gauge will be. On the other hand, reducing `SpaceFraction` to 0 (see [this cookbook example](https://scottplot.net/cookbook/4.1/category/plottable-radialgauge/#gauge-size)) will make the gauge thicker. Could this be a valid workaround for your case?

Adding a new property (`gauges.Width = 50`), just to be used when `RadialGaugeMode.SingleGauge` is set and ignored otherwise, will probably need a reworking of the logic in the `Render` function and might be confusing for the users.

Any thoughts on this @swharden?

**Comment by johndoh** [comment]
I have `SpaceFraction = 0` but I was hoping for something that would make more of a difference. I would like to output something like this:
![Untitled](https://user-images.githubusercontent.com/88682/190595297-01cc2eae-365e-4936-bb3f-2dc6841385af.png)
with  `SpaceFraction = 0` the output is:
![index](https://user-images.githubusercontent.com/88682/190598762-f6c57f59-6781-40ea-8bae-48e4b47e3812.png)

**Comment by johndoh** [comment]
The rendering function has some other limitations which might factor into this feature request. Currently the Radial Gauge does not render correctly with sets of less than 4 or more than 7 data points. This is true for both "normal" mode and Single Gauge Mode.

3 data points - chart bigger than canvas:
![three](https://user-images.githubusercontent.com/88682/190861583-f6573855-c6fc-43b7-94b0-58667fb4f725.png)

4 data points - looks great:
![four](https://user-images.githubusercontent.com/88682/190861596-c6da88cc-67f5-44d4-995f-15257b2cb983.png)

7 data points - looks great:
![seven](https://user-images.githubusercontent.com/88682/190861609-e27cbdac-0d1e-49d3-82a4-77ac96fb46f1.png)

8 data points - chart is smaller than the canvas:
![eight](https://user-images.githubusercontent.com/88682/190861623-34362f00-1adf-4b61-9f91-67a474ae1760.png)

the charts were all generated with the same code from the cookbook, only the number of values is changed.

**Comment by swharden** [comment]
resolved by #2520
